content/tutorial/tutorial.md

```diff
@@ -29,21 +29,21 @@ The tutorial is divided into several sections:
 * [Completing the Game](#completing-the-game) will teach you **the most common techniques** in React development.
 * [Adding Time Travel](#adding-time-travel) will give you **a deeper insight** into the unique strengths of React.
 
-You don't have to complete all of the sections at once to get the value out of this tutorial. Try to get as far as you can -- even if it's one or two sections.
+You don't have to complete all of the sections at once to get the gist out of this tutorial. Try to get as far as you can -- even if it's one or two sections.
```

Comment:
"get the gist of", not "get the gist out of". also, this changes the meaning of the sentence. i think the original is fine